docs/docs/legal/data-processing-agreement.mdx

```diff
@@ -1,113 +1,276 @@
+2) references to specific articles of ‘Regulation (EU) 2016/679’ are replaced with the equivalent article or section of the Swiss FADP,  
+3) references to ‘EU’, ‘Union’ and ‘Member State’ are replaced with ‘Switzerland’,  
+4) Clause 13(a) and Part C of Annex 2 is not used and the ‘competent supervisory authority’ is the Swiss Federal Data Protection Information Commissioner (“**FDPIC**”) or, if the transfer is subject to both the Swiss FADP and the GDPR, the FDPIC (insofar as the transfer is governed by the Swiss FADP) or the DPC (insofar as the transfer is governed by the GDPR),  
+5) references to the ‘competent supervisory authority’ and ‘competent courts’ are replaced with the FDPIC and ‘competent Swiss courts’,  
+6) in Clause 17, the SCCs are governed by the laws of Switzerland,  
+7) in Clause 18(b), disputes will be resolved before the competent Swiss courts, and  
+8) the SCCs also protect the data of legal entities until entry into force of the revised Swiss FADP.
+
+#### UK Transfers
+
+To the extent that Customer Personal Data is subject to Applicable Data Protection Law of the United Kingdom, and the transfer would be a Restricted Transfer, the SCCs as set out above shall apply as amended by Part 2 of the UK Addendum, and Part 1 of the UK Addendum is deemed completed as follows:
+
+1) in Table 1, the details of the parties are set out in the Agreement or, if not set out therein, the applicable statement of work;  
+2) in Table 2, the selected modules and clauses are set out in Section 6.3 of this Addendum;  
+3) in Table 3, the appendix information is set out in the annexes to this DPA or, if not set out therein, the applicable statement of work; and  
+4) in Table 4, the ‘Exporter’ is selected.
+
+#### Alternative Transfer Mechanism
+
+In the event that a court of competent jurisdiction or supervisory authority orders (for whatever reason) that the measures described in this DPA cannot be relied on to lawfully transfer Customer Personal Data, or ZITADEL adopts an alternative data transfer mechanism to the mechanisms described in this DPA, including any new version of or successor to the standard contractual clauses (“Alternative Transfer Mechanism”), the Customer agrees to fully co-operate with ZITADEL to agree an amendment to this Addendum and/or execute such other documents and take such other actions as may be necessary to remedy such non-compliance or give legal effect to such Alternative Transfer Mechanism.
+
+### Additional Provisions under US Data Protection Laws
+
+The Parties agree that all Customer Personal Data that is subject to US Data Protection Laws (including the CCPA) is disclosed to ZITADEL by the Customer for the Permitted Purpose and its use or sharing by the Customer with ZITADEL is necessary to perform such Permitted Purpose.
+
+ZITADEL agrees that it will not:
+
+1. sell or share any Customer Personal Data;  
```

Comment:
Does this need clarification that sell/sharing between our corporate construct is allowed?

Does the sharing part need clarification that we share some data with the 3rd party suppliers we have?